pkg/prometheus/statefulset.go

```diff
@@ -641,7 +641,7 @@ func makeStatefulSetSpec(p monitoringv1.Prometheus, c *operator.Config, shard in
 			return nil, errors.Wrap(err, "failed to build image path")
 		}
 
-		bindAddress := "[$(POD_IP)]"
+		bindAddress := "" // Listen to all available IP addresses by default
```

Comment:
Will it also listen on IPv6 ?

Reply:
According to 
- https://github.com/thanos-io/thanos/blob/ebbd68df68902db404d89d6bd8b3f88d6193d851/pkg/extkingpin/flags.go#L30
- https://github.com/thanos-io/thanos/blob/06ae3b892321b1d4db231aead60df1666b917749/pkg/server/grpc/grpc.go#L117
- https://github.com/thanos-io/thanos/blob/61a6235060a4cb00261726360739d2626c0521ce/pkg/server/grpc/grpc.go#L52 (given to `Net.listen` as first parameter, see second link)
- https://golang.org/pkg/net/#Listen

for grpc, and

- https://github.com/thanos-io/thanos/blob/ebbd68df68902db404d89d6bd8b3f88d6193d851/pkg/extkingpin/flags.go#L47
- https://github.com/thanos-io/thanos/blob/61a6235060a4cb00261726360739d2626c0521ce/pkg/server/http/http.go#L55
- https://golang.org/pkg/net/http/#Server

for http,

Yes.